Sources/FoundationEssentials/Calendar/Calendar_Gregorian.swift

```diff
@@ -1069,20 +1088,24 @@ internal final class _CalendarGregorian: _CalendarProtocol, @unchecked Sendable
 
             case .hour:
                 guard let day = ordinality(of: .day, in: .yearForWeekOfYear, for: date) else { return nil }
-                let hour = dateComponent(.hour, from: date)
+
+                let startOfLarger = start(of: .yearForWeekOfYear, at: date)!
```

Comment:
nit: should this be a part of the `guard` statement above just to defensively protect against `nil` unwrapping here?